src/components/PostsList.tsx

```diff
@@ -2,53 +2,66 @@
 
 import classNames from 'classnames';
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 import { Post } from '../types/Post';
+import * as selectedPostActions from '../features/selectedPost';
+// import { Post } from '../types/Post';
 
-type Props = {
-  posts: Post[];
-  selectedPostId?: number;
-  onPostSelected: (post: Post | null) => void;
-};
+// type Props = {
+//   posts: Post[];
+//   selectedPostId?: number;
+//   onPostSelected: (post: Post | null) => void;
+// };
```

Comment:
These commented-out lines define a `Props` type that is not used. Consider removing them if they are not needed to clean up the code.